packages/engine-formula/src/services/defined-names.service.ts

```diff
@@ -190,6 +214,22 @@ export class DefinedNamesService extends Disposable implements IDefinedNamesServ
     private _update() {
         this._update$.next(null);
     }
+
+    // Update cache
```

Comment:
Some comment is not necessary because they don't provide more information than the method's name.